### ISSUE TITLE
for monterey

### ISSUE THREAD

**Issue by Gavrocheve** [dado3212/spaces-renamer#78]
could you update for monterey version as well thank you ^^

also is the macforge step necessary? will it be unsafe for the system?

**Comment by dumblob** [comment]
It's a bit silly to post this under nearly all open issues in this tracker but here we go.

I had a similar problem and the following sentence I quote (from https://github.com/dado3212/spaces-renamer/issues/74#issuecomment-818411128 ) explained what was the reason (it seems the MacForge as well as spaces-renamer were incorrectly installed for an unknown reason - because in `macOS -> Preferences -> Users & Groups > Login Items` I had only **yellow triangles** for both entries (MacForge as well as spaces-renamer) which means the file the entries were pointing to didn't exist or weren't readable for some reason (permissions?). So I've removed both entries with the ` - ` button (otherwise they could interfere under certain conditions with the repeated manual reinstallation as discussed below).

>If you do not see spaces-renamer in your MacForge plugins then updating names in the spaces-renamer application in your top drawer will not work.

So I followed the recommendation to **not** install spaces-renamer through the "recommendations" in MacForge (I use the latest version) window but **manually** downloaded space-renamer and meticulously followed the steps in the readme in this repo - i.e. first drag & drop `*.bundle` on a running MacForge window, then `killall -9 Dock`, then first double click the `SpacesRenamer.app` file. Then the records in `macOS -> Preferences -> Users & Groups > Login Items` were without yellow triangles and fully working.

And then do **not** be afraid by the button `Update Names` being **greyed out** but just press it anyway!

Feel free to close this issue if it works now for you!
